lab.js

```diff
@@ -116,8 +116,27 @@ const concerns = {
         As -0 === +0 on their own, it may surprise people that they are no longer treated as triple equal when compared
         via a record or tuple. This could lead to bugs.</details>`,
     storingPrimitiveInBox: html`<details><summary>⚠ storing primitives in a Box</summary>
-        If primitives can be stored in a Box, then code checking if a Record/Tuple transitively contains an object can no
-        longer be performed with a containsBoxes predicate. Instead the code will need to also check the contents of the boxes.</details>`,
+        <p>
+            The original rationale for introducing Box is to allow Records and Tuples to explicitly reference a value that would otherwise be disallowed
+            to be 'stored' directly within Records and Tuples e.g. functions.
+            Values like numbers can already be stored in a Record or Tuple.
+            While allowing numbers (and other primitives) to be stored in a Box may be ergonomic for the producer of the Box,
+            complexity has been moved to the consumers of Boxes. Consumers can no longer rely on the guarantee that a Box will always
+            reference a 'non-primitive'.
+        </p>
+        <p>
+            If primitives can be stored in a Box, then code checking if a Record/Tuple transitively contains a mutable object can no
```

Comment:
Technically the problem is not just primitives, but any non identity bearing object, which includes other Box, Record or Tuple.

If the latter are `typeof` `'object'`, we probably should have a new predicate, like `Box.canContain` to make this user check easier. I originally thought of a `Object.isObject` predicate, but it's unclear how that predicate should handle `'function'` which is also not allowed directly in records/tuples.

Reply:
`Object.hasIdentity`?

Reply:
I don't believe that would work in this case since box containing a regular object, and transitively records and tuples containing them also have an identity. I need to make a table with all the different configurations.

Reply:
It should probably say 'values that can be in a Record or Tuple' to avoid any ambiguity, or use of overloaded words like identity. Symbols can be seen as having identity, I can create a symbol that has a unique identity that no one else can forge.

I don't think we need to get the name of the alternative helper methods right here too. Probably better to discuss that in the main R&T repo and then link to that issue from here. What I wanted to capture was that the helper would not be a name that suggests "transitively contains any box", but something else that refers to the values within the boxes.